Dockerfile

```diff
@@ -48,7 +48,8 @@ RUN make build-kfctl
 # Final image base
 #
 
-FROM alpine:3.10.1 as barebones_base
+FROM alpine:$ALPINE_VERSION as barebones_base
+RUN apk add --no-cache ca-certificates  # needed by kfctl
```

Comment:
how did you determine ca-certificates was needed by kfctl?

Reply:
@kkasravi  I'll update the comment, if an https uri is referenced in the Kubeflow app.yaml then Go's http client needs ca-certificates to validate the remote cert. In this case I was testing with the 1.6.2 sample config:

```
apiVersion: kfdef.apps.kubeflow.org/v1alpha1
kind: KfDef
metadata:
  name: kubeflow_app
  namespace: kubeflow
spec:
  repos:
    - name: manifests
      uri: https://github.com/kubeflow/manifests/archive/56e2fb15db286198f7a53723cb1fbfecf3fe83fb.tar.gz
    - name: kubeflow
      uri: https://github.com/kubeflow/kubeflow/archive/0dbd2550372c003ba69069aeee283bd59fb1341f.tar.gz
```